### PR TITLE
Create BLorient11602

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Some new LIT IDs have been created according to our discussions in https://github.com/BetaMasaheft/Documentation/issues/2619, https://github.com/BetaMasaheft/Documentation/issues/2620, https://github.com/BetaMasaheft/Documentation/issues/2621, https://github.com/BetaMasaheft/Documentation/issues/2618. However, the latter ID (LIT7081NamesGod) is still not visible in the app (cf. https://github.com/BetaMasaheft/Works/pull/1222).

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]
thank you

**Review by nafisa-valieva — APPROVED** [review]
Nice!

File: LondonBritishLibrary/orient/BLorient11602.xml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                        <msItem xml:id="ms_i1">
+                            <title ref="LIT7074AynatTsella" type="incomplete"/>
+                            <note>The first two lines written in red containing probably magic characters are mostly illegible.</note>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="illegible"/> በዝ፡ ጠልሰም፡ ወበዝ፡ ክታብ፡ 
+                                አድኅ<supplied reason="undefined" resp="PRS8999Strelcyn">ነ</supplied>ኒ፡ እምሕማመ፡ ዓይነ፡ ጽላ፡ አድኅኖ፡ ለገብረ፡ እግዚአብሔር፡ 
+                                ኪዳነ፡ ማርያም፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <title ref="LIT1763Legend">Magic prayer to Susenyos to protect suckling infants</title>
```
Comment (karljonaskarlsson) [inline LondonBritishLibrary/orient/BLorient11602.xml:40]: Susǝnyos?

**Review by karljonaskarlsson — APPROVED** [review]
Excellent! :)

**Pull request merged**